pkg/services/ngalert/api/test-data/am.http

```diff
@@ -10,23 +10,67 @@ GET http://admin:admin@localhost:3000/alertmanager/{{lokiDatasourceID}}/config/a
 GET http://admin:admin@localhost:3000/alertmanager/{{prometheusDatasourceID}}/config/api/v1/alerts
 
 ###
-# grafana requests
+# create AM configuration
+POST http://admin:admin@localhost:3000/alertmanager/{{grafana}}/config/api/v1/alerts
+content-type: application/json
+
+< ./post-user-config.json
+
+###
+# get latest AM configuration
 GET http://admin:admin@localhost:3000/alertmanager/{{grafana}}/config/api/v1/alerts
+content-type: application/json
 
 ###
+# delete AM configuration
 DELETE http://admin:admin@localhost:3000/alertmanager/{{grafana}}/config/api/v1/alerts
 
 ###
-POST http://admin:admin@localhost:3000/alertmanager/{{grafana}}/config/api/v1/alerts
+# create AM alerts
+POST http://admin:admin@localhost:3000/alertmanager/{{grafana}}/api/v2/alerts
 content-type: application/json
 
-< ./post-user-config.json
+###
+# get silences - no silences
+GET http://admin:admin@localhost:3000/alertmanager/{{grafana}}/api/v2/silences?Filter=foo&Filter=bar
```

Comment:
@gotjosh this request fails with `bad matcher format: foo: unable to list silences`

Reply:
I think that the above error is because of the filter query parameters